cmd/common/image/build.go

```diff
@@ -111,6 +114,15 @@ func NewBuildCmd() *cobra.Command {
 	cmd.Flags().BoolVar(&opts.btfgen, "btfgen", false, "Enable btfgen")
 	cmd.Flags().StringVar(&opts.btfhubarchive, "btfhub-archive", "", "Path to the location of the btfhub-archive files")
 
+	cmd.Flags().BoolVarP(
+		&opts.verifyImage,
+		"verify-image", "",
+		true,
+		"verify container image if policy-controller is installed on the cluster")
```

Comment:
This comment needs to be updated. I got shocked when I read "policy controller" in this context :scream: